packages/patternfly-4/react-core/src/components/AboutModal/AboutModalBoxCloseButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/AboutModalBox/styles.css';
+import { Button } from '../Button';
+import { TimesIcon } from '@patternfly/react-icons';
+
+const propTypes = {
+  /** additional classes added to the About Modal Close button */
+  className: PropTypes.string,
+  /** A callback for when the close button is clicked */
+  onClose: PropTypes.func
+};
+
+const defaultProps = {
+  className: '',
+  onClose: () => undefined
+};
+
+const AboutModalBoxCloseButton = ({ className, onClose, ...props }) => (
+  <div {...props} className={css(styles.aboutModalBoxClose, className)}>
+    <Button variant="plain" onClick={onClose} aria-label="Close Dialog">
```

Comment:
Could import ButtonVariant to use the enum

Reply:
good idea!